src/ui/components/Timeline/Timeline.css

```diff
@@ -129,7 +129,6 @@
 .breakpoint-navigation-timeline .progress-line {
   width: 0%;
   height: 5px;
-  border-radius: 4px;
```

Comment:
this also doesn't seem dark theme related. curious what inspired this

Reply:
Rather than rounding nicely on both the top and bottom, it's only rounding on the top. Probably because the timeline isn't tall enough. I didn't notice it in light theme but it was very noticeable in dark.

Reply:
Got it - i guess we can fix this up in a follow up. It feels boxy now - but probably better than the alternative

Reply:
Agree it feels boxy, yeah. Looking forward to fixing this soon.